src/main/java/edu/cornell/kfs/module/ar/document/service/impl/CuContractsGrantsInvoiceDocumentServiceImpl.java

```diff
@@ -480,8 +480,8 @@
     }
 
     private void setFinalStatusDate(ContractsGrantsInvoiceDocument document, Map<String, Object> parameterMap) {
```

Comment:
As far as I can tell the base code still does not have the else if part on the code that sets the FINAL_STATUS_DATE. Am I missing something?

Reply:
You're right, I don't think the "else if" part is in base code. However, now that KualiCo is using the Approved Date instead of the Finalized Date, we may not need that "else if" portion anymore, since the Approved Date should be ready by the time the PDF gets generated. I would suggest creating a follow-up user story to see if we can remove our customized final-status-date setup.

Reply:
Sounds good, I will create a user story.

Reply:
KFSPTS-15192 has been created.